images/win/toolsets/toolset-2016.json

```diff
@@ -109,7 +109,7 @@
                 "3.1.0",
                 "3.5.0",
                 "3.8.0",
-                "4.3.0"
+                "4.4.0"

```

Comment:
I think such update will be a breaking change:
Previously, folder `C:\Modules\az_4.3.0` existed and customers pointed to it.
After your changes, this folder will be removed and replaced with `C:\Modules\az_4.4.0`.

Reply:
I think we should add it as a separate version